app/components/NotificationIcon.tsx

```diff
@@ -46,15 +46,13 @@ const NotificationIcon: FC = () => {
         MenuListProps={{
           "aria-labelledby": "basic-button",
         }}
-        className={` mt-4 ${open_sans.className}`}
+        className={` mt-4 -ml-24 ${open_sans.className}`}
```

Comment:
there must be a position option on mui to display it on left or right

Reply:
there is actually named anchorOrigin, didn't notice. Do u want me to change it or it's ok?

Reply:
later